packages/dht/src/connection/ConnectorFacade.ts

```diff
@@ -106,26 +106,37 @@ export class DefaultConnectorFacade implements ConnectorFacade {
                 const connectivityResponse = await this.websocketConnector!.checkConnectivity(false)
                 localPeerDescriptor = this.config.createLocalPeerDescriptor(connectivityResponse)
                 this.localPeerDescriptor = localPeerDescriptor
+                this.websocketConnector!.setLocalPeerDescriptor(localPeerDescriptor)
                 if (localPeerDescriptor.websocket === undefined) {
-                    logger.warn('ConnectivityCheck failed after autocertification, websocket server connectivity disabled')
+                    logger.warn('ConnectivityCheck failed after autocertification, disabling websocket server TLS')
```

Comment:
Warnings are visible to the end-users, and therefore could avoid internal names (i.e. `ConnectivityCheck` component)

```suggestion
                    logger.warn('Connectivity check failed after auto-certification, disabling WebSocket server TLS')
```

(similar at line 118)